po/es.po

```diff
@@ -7545,73 +7545,84 @@ msgstr "Te damos la Bienvenida al Día 3"
 
 #: src/welcome-day-3.md
 msgid "Today, we will cover:"
-msgstr ""
+msgstr "Hoy trataremos los siguientes puntos:"
 
 #: src/welcome-day-3.md
 msgid ""
 "Memory management, lifetimes, and the borrow checker: how Rust ensures "
 "memory safety."
 msgstr ""
+"Gestión de la memoria, tiempos de vida y el verificador de préstamos: cómo "
+"garantiza Rust la seguridad de la memoria."
 
 #: src/welcome-day-3.md
 msgid "Smart pointers: standard library pointer types."
-msgstr ""
+msgstr "Punteros inteligentes: tipos de punteros de biblioteca estándar."
 
 #: src/welcome-day-3.md
 msgid "[Welcome](./welcome-day-3.md) (3 minutes)"
-msgstr ""
+msgstr "[Bienvenida](./welcome-day-3.md) (3 minutos)"
 
 #: src/welcome-day-3.md
 msgid "[Memory Management](./memory-management.md) (1 hour and 10 minutes)"
-msgstr ""
+msgstr "[Gestión de la memoria](./Memory-management.md) (1 hora y 10 minutos)"
 
 #: src/welcome-day-3.md
+#, fuzzy
 msgid "[Smart Pointers](./smart-pointers.md) (45 minutes)"
-msgstr ""
+msgstr "[Punteros inteligentes](./smart-pointers.md) (55 minutos)"
 
 #: src/welcome-day-3.md
+#, fuzzy
 msgid ""
 "Including 10 minute breaks, this session should take about 2 hours and 15 "
 "minutes"
 msgstr ""
+"Contando con los descansos de 10 minutos, la duración prevista de la sesión "
+"es de unas 2 horas y 25 minutos."
 
 #: src/memory-management.md
 msgid "[Review of Program Memory](./memory-management/review.md) (5 minutes)"
 msgstr ""
+"[Revisión de la memoria del programa](./Memory-management/review.md) (5 "
+"minutos)"
 
 #: src/memory-management.md
 msgid ""
 "[Approaches to Memory Management](./memory-management/approaches.md) (10 "
 "minutes)"
 msgstr ""
+"[Estrategias para la gestión de la memoria](./Memory-manageme t/approaches."
+"md) (10 minutos)"
 
 #: src/memory-management.md
 msgid "[Ownership](./memory-management/ownership.md) (5 minutes)"
-msgstr ""
+msgstr "[Propiedad](./Memory-management/ownership.md) (5 minutos)"
```

Comment:
```suggestion
msgstr "[Ownership](./Memory-management/ownership.md) (5 minutos)"
```

Let's leave Ownership untranslated. English has like 3 different words used in Rust that mean very different things but they all map to "Propiedad" in spanish.